src/mib_props_supp.c

```diff
@@ -0,0 +1,98 @@
+#include "mib_props_supp.h"
+#include "mib_header_DAC.h"
+#include "mib_header_MQ1.h"
+#include "mib_macros.h"
+#include "mib_utils.h"
+#include "mq1_quad.h"
+#include "mq1_single.h"
+#include "read_mq1_headers.h"
+#include <stdio.h>
+#include <stdlib.h>
+
+/* Mainly this is just for easy access to mq1_fields and dac
+ */
+
+info *mq1_fields_iter(MQ1_fields *fields_struct, size_t *out_count)
```

Comment:
The name suggests it is an iterator but it actually creates an `info` type variable, can we have a better name to communicate the purpose of the function clearer?